### PR TITLE
Work section merge request

### PR DESCRIPTION
**Pull request by glitcher255**
Created full work section.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
 
     <base1 id="base1"></base1>
 
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
+    <img src="./assets/Vector.svg" alt="">
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

File: index.html [hunk 2]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
+      <me_secondary_text><p id="me_secondary_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></me_secondary_text>
+      <a href="https://github.com/glitcher255/micro-porfolio"><me_btn class="orange_btns" id="me_btn"><p id="me_btn_text">Get My Resume</p></me_btn></a>
+      <me_art id="me_art"></me_art>
+      <me_rectangle id="me_rectangle"><img src="./assets/Rectangle 73.png" alt=""></me_rectangle>>
+      <me_vector id="me_vector"><img src="./assets/Vector 4.png" alt=""></me_vector>
+    </about_me>
+
+  <skill id="skill">
+    <ul id="frame30">
+      <frame30_rectangle id="frame30_rectangle"></frame30_rectangle><p id="language_text">language</p></frame30_rectangle>
```
Comment (mgmediaweb) [inline index.html]: Kindly suggest that when you use UL/LI listings, just put LI listings, remove other tags

File: index.html [hunk 3]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.

File: index.html [hunk 4]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.

File: index.html [hunk 5]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.

File: index.html [hunk 6]
```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @glitcher255 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

work4

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 

## Required Changes ♻️

[NOTE] I'm going to put some corrections that should be on the code, but can't in this case.

- Kindly, I suggest you to change some tags you put by semantic valid HTML tags, or DIV. I mean item_nav, fram4, fram3, fram5, johndoe, normal_btn (there are too many, please read the HTML/CSS documentation)

- Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image.

- Kindly, I suggest you to remove the above image that shows the phone information.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (glitcher255) [reply index.html]: This breaks formatting and is not a part of this project, this is extra code for next week's project, also can you please clarify a bit more what you mean by other tags?

File: index.html (hunk 3 above)
Comment (glitcher255) [reply index.html]: Grammer issues in this feedback, I assume you mean change action_btn to button? in which case I have done so.

File: index.html (hunk 1 above)
Comment (glitcher255) [reply index.html]: Done.

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html]: Only LI inside UL lists.
https://www.w3schools.com/html/html_lists.asp

File: index.html (hunk 3 above)
Comment (mgmediaweb) [reply index.html]: you need to put a link here, you can put a button or an "A" link

File: index.html (hunk 2 above)
Comment (glitcher255) [reply index.html]: This simply breaks my code, so I just removed it as its not required for now.

File: index.html (hunk 3 above)
Comment (glitcher255) [reply index.html]: Done.

**Comment by glitcher255** [comment]
@mgmediaweb Implemented all your changes and pushed to github, let me know if there is anything else needed before I request another review.

File: index.html [hunk 7]
```diff
@@ -9,42 +9,42 @@
...
-            </fram4>
-            <fram3>
+            </div>
+            <div>
                 <about id="about"><p>About</p></about>
-            </fram3>
-            <fram5>
+            </div>
+            <div>
                 <contact id="contact"><p>Contact</p></contact>
```
Comment (ha-manel) [inline index.html]: - [x] As mentioned by the previous reviewer, kindly use valid HTML5 tags. There is no tag called `portf` or `about` or `contact`...etc 👍kindly refer to [this](https://www.w3schools.com/TAGS/default.asp)

File: index.html [hunk 8]
```diff
@@ -80,5 +80,164 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
...
+      </div>
+
+    <div id="about_me">
+      <me_primary_text><p id="me_primary_text">About me</p></me_primary_text>
+      <me_secondary_text><p id="me_secondary_text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></me_secondary_text>
+      <a href="https://github.com/glitcher255/micro-porfolio"><me_btn class="orange_btns" id="me_btn"><p id="me_btn_text">Get My Resume</p></me_btn></a>
+      <me_art id="me_art"></me_art>
+      <me_rectangle id="me_rectangle"><img src="./assets/Rectangle 73.png" alt="Rectangle"></me_rectangle>>
+      <me_vector id="me_vector"><img src="./assets/Vector 4.png" alt="Divider"></me_vector>
+    </div>
```
Comment (ha-manel) [inline index.html]: - [x] Kindly put back the section about the skills, it is part of the requirements as you can see they put it for template 1 but it applies to the other templates too. Please use the structure mentioned by the previous reviewer 👍 

## Project requirements:
image
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @glitcher255 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Github flow followed correctly ✔️ 

## Required Changes:
_Kindly check the comments below_

## Optional:
- [x] Please consider using best practices in writing the commit messages, they should be capitalized sentences in the imperative form. Example: "Fix button styling" instead of "fixed button styling" or "button styling fixed"...
- [x] Please consider adding the MIT file, the link in your README file is broken 👍 

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 7 above)
Comment (glitcher255) [reply index.html]: The previous owner did not tell me to change elements in pre approved earlier projects, I changed those elements now.

File: index.html (hunk 8 above)
Comment (glitcher255) [reply index.html]: Finished.

**Comment by glitcher255** [comment]
Implemented all changes, submitting for hopefully the last review now.

File: index.html [hunk 9]
```diff
@@ -9,76 +9,259 @@
...
         
-        <illustration id="illustration">
-        </illustration>
+        <div id="illustration">
+        </div>
+
+    <div id="base1"></div>
+
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
```
Comment (mgmediaweb) [inline index.html]: Kindly, in previous reviews I told you just to use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm

File: index.html [hunk 10]
```diff
@@ -9,76 +9,259 @@
...
+          <div id="card_works">
+            <div id="action">
+              <img src="./assets/Mask Group.png" alt="Professional art data">
+            </div>
+            <a href="https:github.com"><div id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </div></a>
+
+            <li id="card_data">
+              <div id="card_title"><p>Profesional Art Printing Data</p></div>
+              <div id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
```
Comment (mgmediaweb) [inline index.html:116]: @@@@

File: index.html [hunk 11]
```diff
@@ -9,76 +9,259 @@
...
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
```
Comment (mgmediaweb) [inline index.html]: Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm

File: index.html [hunk 12]
```diff
@@ -9,76 +9,259 @@
...
```
Comment (mgmediaweb) [inline index.html]: Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm

File: index.html [hunk 13]
```diff
@@ -9,76 +9,259 @@
...
+        <li class="frame30_text"><p class="frame30_font">
+        <li class="frame30_text"><p class="frame30_font">Codekit</p></li>
+        <li class="frame30_text"><p class="frame30_font">Github</p></li>
+        <li class="frame30_text"><p class="frame30_font">Codepen</p></li>
+        <li class="frame30_text"><p class="frame30_font">Gitlab</p></li>
+        <li class="frame30_text"><p class="frame30_font">Terminal</p></li>
+        </ul>
+    </ul>
+    </div>
+</recentwork>
```
Comment (mgmediaweb) [inline index.html]: Kindly, in previous reviews I told you only use valid HTML labels, and turn them into correct or divs, I attached documentation about HTML and its correct uses. (Check all your code to remove them all) https://www.tutorialspoint.com/html5/html5_tags.htm

File: index.html [hunk 14]
```diff
@@ -9,76 +9,259 @@
...
+      <li id="frame30">
+        <div id="circle_rectangle"></div><p id="language_text">Skills</p>
+        <ul>
+        <li class="frame30_text"><p class="frame30_font">
+        <li class="frame30_text"><p class="frame30_font">Codekit</p></li>
+        <li class="frame30_text"><p class="frame30_font">Github</p></li>
+        <li class="frame30_text"><p class="frame30_font">Codepen</p></li>
+        <li class="frame30_text"><p class="frame30_font">Gitlab</p></li>
+        <li class="frame30_text"><p class="frame30_font">Terminal</p></li>
+        </ul>
```
Comment (mgmediaweb) [inline index.html]: Kindly, I suggest checking your UL/LI lists structures, they lack closing labels or you put "div" tags that do not correspond (This comment apply it in all your UL/LI lists of the page, check it well!), I leave you a tutorial of how to use them correctly. https://www.w3schools.com/html/html_lists.asp

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @glitcher255 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost4

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 12 above)
Comment (glitcher255) [reply index.html]: can you clarify what the wrong HTML label here is?

File: index.html (hunk 10 above)
Comment (glitcher255) [reply index.html:116]: what is the wrong label here?

**Comment by glitcher255** [comment]
what a nightmare.

File: index.html (hunk 14 above)
Comment (glitcher255) [reply index.html]: You need to be a little more specific here, what exactly did I close with /div??

Comment (glitcher255) [reply index.html]: @mgmediaweb

**Comment by glitcher255** [comment]
Fixed everything requested that I could find, please be more specific in your reviews

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @glitcher255 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — COMMENTED** [review]
Hello @glitcher255 👋,

_This is an additional review as a result of the second opinion request._

Looks like your project is already approved, and your issue has resolved itself. 


Keep rocking 🚀!

**Pull request merged**